teuthology/task/knfsd.py

```diff
@@ -104,6 +104,29 @@ def task(ctx, config):
                 '{MNT}'.format(MNT=mnt),
                 ],
             )
+
+        """
+        Start NFS kernel server
+        """
+        args=[]
+        if remote.os.package_type == 'rpm':
+            args=[
+                'sudo',
+                "service",
+                'nfs',
+                'restart',
+                ]
+        else:
+            args=[
+                'sudo',
+                "service",
+                'nfs-kernel-server',
```

Comment:
This could be abstracted a bit; both codepaths use ``sudo service NAME restart``